Library/Homebrew/cmd/update.sh

```diff
@@ -745,6 +745,20 @@ EOS
     fi
   done
 
+  if [[ -n "${HOMEBREW_INSTALL_FROM_API}" ]]
+  then
+    mkdir -p "${HOMEBREW_CACHE}/api"
+    # TODO: etags?
```

Comment:
```suggestion
# TODO: use --header If-Modified-Since
```

Reply:
I figured for now it's okay since we use `If-Modified-Since` when we download in `Homebrew::API::Formula::all_formulae` but we can add it here too in the future with the rest of the `brew update` changed that we'll need to do

Reply:
Perfect!